react/features/mobile/external-api/middleware.js

```diff
@@ -429,3 +437,30 @@ function escapeBackslashes(value: string, alsoOnIos?: boolean) {
         ? value.replace(/\\/g, '\\\\')
         : value;
 }
+
+function getRemoteDescriptions(store) {
+    const connection = store.getState()['features/base/connection'].connection;
+
+    if (!connection || !connection.xmpp || !connection.xmpp.getSessions) {
+        return undefined;
+    }
+    const sessions = Object.values(connection.xmpp.getSessions());
+
+    const remoteDescriptions
+        = sessions
+            .map(session => {
+                if (!session.peerconnection || !session.peerconnection.remoteDescription) {
+                    return null;
+                }
+
+                return {
+                    'localJid': session.localJid,
+                    'remoteJid': session.remoteJid,
+                    'type': session.peerconnection.remoteDescription.type,
+                    'sdp': session.peerconnection.remoteDescription.sdp
+                };
+            })
+            .filter(rd => Boolean(rd));
+
+    return remoteDescriptions && remoteDescriptions.length ? remoteDescriptions : undefined;
```

Comment:
Returning null or undefined is always a bad idei, because it can make an unexpected crash if shomewere there are no handlings. Return an empty object or empty array or empty string is always a better idea. Please change every return null and return undefined.